runtime/trappist-common/src/xcm_config.rs

```diff
@@ -0,0 +1,121 @@
+use crate::impls::AccountIdOf;
```

Comment:
At the moment we have https://github.com/paritytech/trappist/tree/main/primitives/xcm which seems to fill the same need. I wonder if we should move these to the primitives folder or if primitives should be moved here. This convention comes from Statemine as well.